src/components/fields/select-histogram/types.ts

```diff
@@ -0,0 +1,14 @@
+import { SelectHistogramProps } from "@lifesg/react-design-system";
+import { TComponentOmitProps } from "../../frontend-engine";
+import { IHistogramValidationRule } from "../histogram-slider";
+import { IBaseFieldSchema } from "../types";
+
+type TCustomOptions = Pick<SelectHistogramProps, "rangeLabelPrefix" | "rangeLabelSuffix">;
+
+export interface ISelectHistogramSchema<V = undefined>
+	extends IBaseFieldSchema<"select-histogram", V, IHistogramValidationRule>,
+		TComponentOmitProps<
+			Pick<SelectHistogramProps, "className" | "disabled" | "readOnly" | "value" | "histogramSlider">
```

Comment:
instead of picking the entire `histogramSlider`, just pick `bins` and `interval` will do

Reply:
can dont pick `histogramSlider` anymore since we are declaring it manually
```suggestion
			Pick<SelectHistogramProps, "className" | "disabled" | "readOnly" | "value">
```